packages/react-core/src/components/Avatar/__tests__/Avatar.test.tsx

```diff
@@ -1,8 +1,20 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import {render, screen } from '@testing-library/react';
 import { Avatar } from '../Avatar';
 
-test('simple avatar', () => {
-  const { asFragment } = render(<Avatar alt="avatar" src="test.png" border="light" />);
-  expect(asFragment()).toMatchSnapshot();
+test('renders simple avatar', () => {
+  render(<Avatar alt="avatar" data-testid="avatar" src="test.png" border="light" />);
+  expect(screen.getByTestId('avatar')).toBeVisible();
 });
+
+test('renders with class name pf-m-light when "light" is passed as border prop', () => {
+  render(<Avatar alt="avatar" data-testid="avatar" src="test.png" border="light" />);
+  expect(screen.getByTestId('avatar')).toHaveClass('pf-m-light');
+});
```

Comment:
I've gone back and forth a bit with how I feel about testing the different values that can be provided for a prop.

In situations like this prop where the different options are very limited and different classes are applied based on the option used I think each option (and the default behavior) should be tested, i.e. a test that neither class is applied by default, a test that the light class is applied with the light option, and a test that the dark class is applied with the dark option.